lib/matplotlib/mlab.py

```diff
@@ -2867,14 +2867,19 @@ def get_func(name, item, func):
         'print':  'print_',
         }
 
-    def get_converters(reader):
+    def get_converters(reader, comments):
 
         converters = None
-        for i, row in enumerate(reader):
+        i = 0
+        for row in reader:
+            if (len(row) and comments is not None and
+                    row[0].startswith(comments)):
```

Comment:
hmmm, we are only checking a single character, so maybe it might make more sense to do `row[0] in comments`, which would eliminate the tuple restriction noted above.